server/routes/ticket.py

```
@@ -302,5 +300,333 @@ def post(self):
+                    )
+
+            db.session.commit()
+
+            return restful_response(
+                status="success",
+                message="Ticket updated successfully",
+                data=ticket.to_dict(),
+                status_code=200,
+            )
+
+        except Exception as e:
```

Flake8 (7.3.0)

[error] local variable 'e' is assigned to but never used

(F841)